src/chapters/jdbc-auth.xml

```diff
@@ -118,6 +225,135 @@
+                                                    <para>Require SSL connections, and
+                                                        verify that the server certificate
+                                                        is issued by a known certificate
+                                                        authority.</para>
+                                                </listitem>
+                                            </varlistentry>
+                                            <varlistentry>
+                                                <term>verify-full</term>
+                                                <listitem>
+                                                    <para>Require SSL connections, 
+                                                        verifying that the server certificate
+                                                        is issued by a known authority, and
+                                                        that the name on the certificate
+                                                        matches the name of the server.</para>
+                                                </listitem>
+                                            </varlistentry>
+                                        </variablelist>
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-cert-file</property></entry>
+                                    <entry>
+                                        <para>The client certificate file to use when making
+                                            an SSL connection to the Postgres server. This
+                                            property is optional, and will be ignored if
+                                            the SSL mode is set to disable.</para>
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-key-file</property></entry>
+                                    <entry>
+                                        <para>The file containing the client private key
+                                            to be used when making an SSL-encrypted connction
+                                            to a Postgres server.  This property is optional
+                                            and will be ignore if the SSL mode is set to
+                                            disable.</para>
+                                    </entry>
+                                </row>
+                                <row>
+                                    <entry><property>postgresql-ssl-root-cert-file</property></entry>
+                                    <entry>
+                                        <para>The file that contains the root certificates
```

Comment:
And here - it's unclear from the description what type of file this property should point to.